modules/Contacts/Contact.php

```diff
@@ -186,6 +187,11 @@ public function Contact()
         }
         self::__construct();
     }
+    
+        
+    public function getEmailBean() {
```

Comment:
The PHPDocs seem to be missing

Reply:
in the interface but I can add it here also